### ISSUE TITLE
Add MonadUnliftIO instances

### ISSUE THREAD

**Issue by phadej** [scrive/log#43]
we'll need to depend on `unliftio-core` which is small package.

Note: I **don't** want to change 
```haskell
instance (MonadBase IO m, MonadTime m) => MonadLog (LogT m)
```

as I sometimes use `LogT` with `ExceptT` or `StateT`, which cannot be made into `MonadUnliftIO` (otherwise I'll need to use exceptions and `IORef`s, do not like)

But newer `ResourceT` wants things to be `MonadUnliftIO` (and honestly, I'm lazy commuting `Reader` like transformers in my stacks, so I'd rather add an instance).

*EDIT* I'll make a PR if change is OK. I have the code already.

**Comment by 23Skidoo** [comment]
OK sure, why not.

**Comment by 23Skidoo** [comment]
This was fixed by #47.
